Lib/test/test_clinic.py

```diff
@@ -3194,8 +3194,12 @@ def test_py_ssize_t_converter(self):
             ac_tester.py_ssize_t_converter(PY_SSIZE_T_MAX + 1)
         with self.assertRaises(TypeError):
             ac_tester.py_ssize_t_converter([])
-        self.assertEqual(ac_tester.py_ssize_t_converter(), (12, 34, 56))
-        self.assertEqual(ac_tester.py_ssize_t_converter(1, 2, None), (1, 2, 56))
+        with self.assertRaises(ValueError):
```

Comment:
Add a parser test that checks that `allow_negative` cannot be combined with other types of converters. And also check what happens when we specify a negative default value with `allow_negative=False` (whether it's a default C value or a default Python value)

Reply:
> And also check what happens when we specify a negative default value with allow_negative=False (whether it's a default C value or a default Python value)
I've added a test including a default negative c value. I don't quite understand what you mean with a default Python value.

Reply:
We can have `py_default` and `c_default` in an AC converter, I assume this is what Bénédikt meant.